### PR TITLE
[Docs] Adding move-analyzer VS Code plugin link.

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Checking this simple task off my list.

cc: @geekflyer

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/1772)

### CODE REVIEW

File: developer-docs-site/docs/guides/getting-started.md [hunk 1]
```diff
@@ -61,9 +61,10 @@ Start by cloning the `aptos-core` GitHub repo from [[GitHub](https://github.com/
 
 Now your basic Aptos development environment ready.
 
-## Plugins
+## Move plugins
```
Comment (geekflyer) [inline developer-docs-site/docs/guides/getting-started.md:64]:
```suggestion
## IDE plugins for Move
```

File: developer-docs-site/docs/guides/getting-started.md [hunk 2]
```diff
@@ -61,9 +61,10 @@ Start by cloning the `aptos-core` GitHub repo from [[GitHub](https://github.com/
...
 
 - A [Move language syntax plugin](https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax) for Visual Studio Code.
+- A [Visual Studio Code plugin for Move, called move-analyzer](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer).
```
Comment (geekflyer) [inline developer-docs-site/docs/guides/getting-started.md:67]:
```suggestion
- [Move-Analyzer for VS Code](https://marketplace.visualstudio.com/items?itemName=move.move-analyzer) - supports advanced code navigation and syntax highlighting
```

File: developer-docs-site/docs/guides/getting-started.md [hunk 3]
```diff
@@ -61,9 +61,10 @@ Start by cloning the `aptos-core` GitHub repo from [[GitHub](https://github.com/
 
 Now your basic Aptos development environment ready.
 
-## Plugins
+## Move plugins
 
 - A [Move language syntax plugin](https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax) for Visual Studio Code.
```
Comment (geekflyer) [inline developer-docs-site/docs/guides/getting-started.md:66]:
```suggestion
- [Syntax Hightlighting for VS Code](https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax)
```

File: developer-docs-site/docs/guides/getting-started.md (hunk 2 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/guides/getting-started.md:67]: The term `move-analyzer` is the name of the plugin, and we should use the term as it is. Changing the above suggested `Move-Analyzer` to `Move analyzer` so now this is just a label.

**Pull request merged**